src/components/Title/styled.ts

```diff
@@ -0,0 +1,42 @@
+import styled, {css} from "styled-components";
+
+interface props {
+  variant: string;
+}
+
+export const TitleWrapper = styled.div<props>`
+  justify-content: center;
+  text-align: center;
+  align-items: center;
+  width: fit-content;
+  max-width: 500px;
+  flex-direction: row;
+  display: flex;
+  font-family: 'Chakra Petch';
+  font-style: italic;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 42px;
+  :before{
+    height: 46px;
+    margin-right: 18px;
+    content: '';
+    width: 5px;
+    background-color: ${({ theme }) => theme.colors.primary};
+    position: relative;
+    display: inline-block;
+  }
+
+  ${({variant}) => variant == 'secondary' ? css`
+    :after {
+      height: 46px;
+      margin-left: 18px;
+      content: '';
+      width: 5px;
+      background-color: ${({ theme }) => theme.colors.primary};
+      position: relative;
+      display: inline-block;
+    }
+  
+  `: null };
```

Comment:
Pra evitar colocar o `null` no final, eu sugiro isso
```suggestion
  ${({variant}) => variant == 'secondary' && css`
    :after {
      height: 46px;
      margin-left: 18px;
      content: '';
      width: 5px;
      background-color: ${({ theme }) => theme.colors.primary};
      position: relative;
      display: inline-block;
    }
  
  `};
```